example_envs/bookinfo/gateway.rs

```diff
@@ -32,6 +32,8 @@ impl SimElement for Gateway {
                 .insert("direction".to_string(), "request".to_string());
             rpc.headers
                 .insert("src".to_string(), self.core_node.id.to_string());
+            rpc.headers
+                .insert("dest".to_string(), "productpage-v1".to_string());
```

Comment:
I do not think dest is needed? This is already part of the return tuple. We should choose one of them.

Reply:
Okay, I will get rid of it here.

Reply:
I think we should either use "dest" or target. Right now all of this is redundant and it can easily lead to mistakes. I will make a pull request some time on this.